ui/package.json

```diff
@@ -17,9 +17,10 @@
     "create:fragment-matcher": "dotenv node src/scripts/queryServer"
   },
   "dependencies": {
-    "@material-ui/core": "^3.9.3",
+    "@material-ui/core": "^4.2.0",
     "@material-ui/icons": "^3.0.2",
     "@material-ui/lab": "3.0.0-alpha.30",
```

Comment:
`@material-ui/lab` could probably also be bumped to the alpha version of v4. We use a couple of components from there. I hope there isn't anything critical to change. The CHANGELOG should be able to say what changed.